src/components/common/EthHashInfo/index.tsx

```diff
@@ -74,6 +74,10 @@ const EthHashInfo = ({
             {showPrefix && shouldPrefix && prefix && <b>{prefix}:</b>}
             {shortAddress ? shortenAddress(address) : address}
           </Typography>
+          <Typography variant="body2" fontWeight="inherit" component="div" className={css.mobileAddress}>
+            {showPrefix && shouldPrefix && prefix && <b>{prefix}:</b>}
+            {shortenAddress(address)}
```

Comment:
I would add only this part instead of the whole Typography. It could be wrapped in a span.

Reply:
Makes sense, thanks!